recipes/chamois/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "CHAMOIS" %}
+{% set name_pypi = "chamois-tool" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.org/packages/source/c/{{ name_pypi }}/{{ name_pypi | replace('-', '_') }}-{{ version }}.tar.gz
+  sha256: 4812d2209b3867236892197348ce9c6ddecf7e7444207d4ced10a2068d898b8e
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  run_exports:
+    - {{ pin_subpackage("chamois", max_pin="x.x") }}
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - setuptools >=39.2
+    - wheel >=0.30
+  run:
+    - python >=3.7
+    - importlib_resources >=1.0  #[py37, py38, py39]
+    - numpy >=1.16
+    - anndata >=0.8
+    - gb-io >=0.3.1
+    - lz4 >=4.0
+    - numpy >=1.0
+    - pandas >=1.3
+    - platformdirs >=3.0
+    - pyhmmer >=0.11.0
+    - pyrodigal >=3.0
+    - rich >=12.4.0
+    - rich-argparse >=1.1
+    - scipy >=1.4
+    - scikit-learn >=1.0
+    - kennard-stone >=2.1
+    - rdkit >=2023.3
```

Comment:
do we need these if extras are not being installed (or are they)?

Reply:
If extras are not installed with `pip install` but present they are still used -- these two dependencies are needed for the `train` command.